app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/TrackerBlockingVpnService.kt

```diff
@@ -232,8 +231,10 @@ class TrackerBlockingVpnService : VpnService(), CoroutineScope by MainScope() {
                 return@withContext
             }
 
-            // We need to rethink how to log this state. This will likely change.
```

Comment:
Looks like I can't add suggestions that are too far from code changes, but here is another issue that @aitorvs and I uncovered: on line 228 - we need to change `if (currStateStats?.state == ENABLING) {` to `if (currStateStats?.state == ENABLING || currStateStats?.state == ENABLED) {`. This is an improvement on my change in https://github.com/duckduckgo/Android/pull/2637.